code/lib/cli/src/sandbox-templates.ts

```diff
@@ -443,7 +443,7 @@ const baseTemplates = {
   },
   'qwik-vite/default-ts': {
     name: 'Qwik CLI Latest (Vite | TypeScript)',
-    script: 'yarn create qwik basic {{beforeDir}}',
+    script: 'npm create qwik basic {{beforeDir}}',
```

Comment:
is this a safe change? I wonder if this will work given that it will end up generating npm lock files, and further down the sandbox pipeline things might not work. Can you trigger the sandbox generation for this branch?

I honestly don't remember why we use Yarn 1 in the sandboxes, might be worth experimenting switching the Yarn version as well, maybe

Reply:
> is this a safe change? I wonder if this will work given that it will end up generating npm lock files, and further down the sandbox pipeline things might not work. Can you trigger the sandbox generation for this branch?

Did a temporary run here, looks like it worked fine. https://github.com/storybookjs/storybook/actions/runs/8171749412/job/22340650305#step:11:221

> I honestly don't remember why we use Yarn 1 in the sandboxes, might be worth experimenting switching the Yarn version as well, maybe

It was probably the easiest, but I think it's pretty bad now, especially because StackBlitz doesn't like it too much (our StackBlitz runs `npm install && yarn storybook` 🙃)

I think the right thing to do here is to migrate sandboxes to npm - we don't need to prescribe users a certain package manager so the default should be fine. I tried this some month ago but weren't successful in a small timebox, I wanna try again though.